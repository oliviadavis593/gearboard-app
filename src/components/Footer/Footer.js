import React from 'react';
import '../../styles/Footer.css'


export default function Footer() {
    return(
        <footer className='footer'>
                <div className='footer__brand-container'>
                    <p className='footer__author-text'>
                        Created by <a href='https://oliviadavis593.github.io/portfolio/' target='_blank' rel="noopener noreferrer">Olivia Davis</a>
                    </p>
                </div>
                <div className='footer__social-media'>
                    <a href='https://github.com/oliviadavis593' target='_blank' rel="noopener noreferrer">GitHub</a>
                    <a href='https://www.linkedin.com/in/olivia-davis-ab1848105/' target='_blank' rel="noopener noreferrer">LinkedIn</a>
                </div>
                <div className='copyright-text'>
                    <p>
                        Copyright &copy; 2020
                    <br />
                        All Rights Reserved
                    </p>
                </div>
            </footer>
    )
}