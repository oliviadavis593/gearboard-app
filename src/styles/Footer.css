.footer {
    text-align: center;
    width: 100%;
    color: #fff;
  }

.footer__author-text a {
  color: #000;
}

.footer__author-text:hover a{
  color: #e67e22;
  transition: 0.5s;
}