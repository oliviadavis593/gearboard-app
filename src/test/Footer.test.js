import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer/Footer';


describe('Footer component', () => {
    it(`renders without crashing`, () => {
        const div = document.createElement('div')
        ReactDOM.render(<Footer />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<Footer />)
            .toJSON();
    expect(tree).toMatchSnapshot();
    })
})