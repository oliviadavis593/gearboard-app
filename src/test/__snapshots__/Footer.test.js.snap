// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders the UI as expected 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__brand-container"
  >
    <p
      className="footer__author-text"
    >
      Created by 
      <a
        href="https://oliviadavis593.github.io/portfolio/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Olivia Davis
      </a>
    </p>
  </div>
  <div
    className="footer__social-media"
  >
    <a
      href="https://github.com/oliviadavis593"
      rel="noopener noreferrer"
      target="_blank"
    >
      GitHub
    </a>
    <a
      href="https://www.linkedin.com/in/olivia-davis-ab1848105/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LinkedIn
    </a>
  </div>
  <div
    className="copyright-text"
  >
    <p>
      Copyright Â© 2020
      <br />
      All Rights Reserved
    </p>
  </div>
</footer>
`;
